.circleci/config.yml

```diff
@@ -50,6 +50,14 @@ references:
     branches:
       ignore: /.*/
 
+filters_only_renovate: &filters_only_renovate
+    branches:
+      only: /^renovate-.*/
+
+filters_only_nori: &filters_only_nori
+    branches:
+      only: /^nori/.*/
```

Comment:
since these trigger the same jobs in their workflows, could we group the regexes and remove the duplication?